### PR TITLE
Make borgs shown in logprobe

### PR DESCRIPTION
**Pull request by lzk228**
## About the PR

If entity accessed by borg - logprobe will show their name identifier

## Why / Balance

dont know

## Media

Screenshot_20240508_041033

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl:
- tweak: Borgs ID now will be shown in LogProbe instead of Unknown.

### CODE REVIEW

**Comment by lzk228** [comment]
I just thought, would it be better to use only name indentifier in case if in future not only borgs will be able to access doors

File: Content.Shared/Access/Systems/AccessReaderSystem.cs [hunk 1]
```diff
@@ -399,6 +401,9 @@ private void LogAccess(Entity<AccessReaderComponent> ent, EntityUid accessor)
             && idCard.Comp is { BypassLogging: false, FullName: not null })
             name = idCard.Comp.FullName;
 
+        if (TryComp<NameIdentifierComponent>(accessor, out var nameIdentifier) && HasComp<BorgChassisComponent>(accessor))
+        name = nameIdentifier.FullIdentifier;
+
```
Comment (AJCM-git) [inline Content.Shared/Access/Systems/AccessReaderSystem.cs:406]: the indentation of name is wrong, also put this check before the idCard one as that one takes priority

I also see no reason to limit this to borgs so remove that check

**Review by AJCM-git — CHANGES_REQUESTED** [review]

File: Content.Shared/Access/Systems/AccessReaderSystem.cs [hunk 2]
```diff
@@ -399,6 +401,9 @@ private void LogAccess(Entity<AccessReaderComponent> ent, EntityUid accessor)
             && idCard.Comp is { BypassLogging: false, FullName: not null })
             name = idCard.Comp.FullName;
 
+        if (TryComp<NameIdentifierComponent>(accessor, out var nameIdentifier))
+            name = nameIdentifier.FullIdentifier;
+
```
Comment (AJCM-git) [inline Content.Shared/Access/Systems/AccessReaderSystem.cs:406]: remember i said you should put this check before the idCard one

**Review by AJCM-git — CHANGES_REQUESTED** [review]

File: Content.Shared/Access/Systems/AccessReaderSystem.cs (hunk 2 above)
Comment (lzk228) [reply Content.Shared/Access/Systems/AccessReaderSystem.cs]: oh yes, makes sense

**Review by AJCM-git — APPROVED** [review]

**Pull request merged**